.github/workflows/release.yaml

```
@@ -0,0 +1,207 @@
+          tags: |
+            ghcr.io/arkade-os/arkd-wallet:${{ github.event.release.tag_name }}
+            ghcr.io/arkade-os/arkd-wallet:latest
+          platforms: linux/amd64,linux/arm64
+          build-args: |
+            VERSION=${{ github.event.release.tag_name }}
+
+      - name: Build and push arkd Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ./Dockerfile 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)